x-pack/plugins/enterprise_search/server/routes/app_search/search_settings.ts

```diff
@@ -64,8 +48,9 @@ export function registerSearchSettingsRoutes({
         params: schema.object({
           engineName: schema.string(),
         }),
-        body: searchSettingsSchema,
+        body: schema.buffer(),
       },
+      options: { body: { parse: false } },
```

Comment:
I believe these options are just pass-through to hapi. You can read about the options here: https://hapi.dev/api/?v=20.1.2.

By default, the `output` option is 'data'. Here's what it has to say about that:

> the incoming payload is read fully into memory. If parse is true, the payload is parsed (JSON, form-decoded, multipart) based on the 'Content-Type' header. If parse is false, a raw Buffer is returned.

Since we set `parse` to false, the body content comes through as a Buffer, and is *not* parsed to JSON.

Reply:
Would love a link to those docs in the docblock comment once we have a helper for this!

Reply:
I wasn't *entirely* sure whether these are actually 1 to 1 or not ... though I strongly suspect they are, but because of the uncertainty I decided to leave them out. Folks probably should rely on Kibana server docs I'm thikning. I can add them in if you prefer though

Reply:
Nah, Kibana docs are perfect as well!